### PR TITLE
Bug 1905347: Gherkin Scripts Scenarios epic ODC-1052

### PR DESCRIPTION
**Pull request by gajanan-more**
Fixes: https://issues.redhat.com/browse/ODC-4881

Problem: Gherkin Scripts for scenarios for [ODC-1052](https://issues.redhat.com/browse/ODC-1052)

Solution: These are the gherkin scripts designed covering all the scenarios related to epics [ODC-1052](https://issues.redhat.com/browse/ODC-1052)

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Feature: Create Application from git form
+    As a user, I want to deploy an application from git repo with devfile
+
+
+    Background:
+        Given user is at developer perspective
+        And user is at Add page
```
Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature:7]: You can remove this from here as it varies in the scenarios

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        When user clicks on From Devfile card
+        And user enters Git Repo url as "https://github.com/maysunfaisal/node-bulletin-board"
+        And user enters Name as "node-bulletin-board"
+        And user clicks on Create button
+        Then user will be redirected to Topology page
+        And user is able to see workload "node-bulletin-board" in topology page
+
+
+    @regression
+    Scenario: Deploy new application from git repo with devfile from topology page
```
Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature:29]: Won't there be an edit scenario option?

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature (hunk 1 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature:7]: Updated

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature (hunk 2 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature:28]: I have added the edit scenario but it's incomplete. There is a bug raised and we are unaware of the feature. Once the bug is resolved, we will update the scenario. We reached to this conclusion after discussing it with Andrew.

Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature:28]: Do have a task assigned in your name for  it as and when the bug is fixed you can include the changes in the script

Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature:28]: Not yet. I will create one when the bug will get fixed.

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        And user is able to see workload "node-bulletin-board" in topology page
+
+
+#Incomplete as bug raised related to this feature.
+    @regression
+    Scenario: Edit application deployed using From Devfile option
+        Given user has created workload "node-bulletin-board" using From Devfile option
+        And user is at topology page
+        When user opens context menu for workload "node-bulletin-board"
+        And user selects "Edit node-bulletin-board" option from context menu
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature:47]: Not for DevPreview.

Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature:47]: The incomplete scenario will be completed once the feature is in.

Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature]: Removed that scenario.

**Pull request merged**